### PR TITLE
"Added alokVema749 in LinkFree"

### PR DESCRIPTION
**Pull request by alokVerma749**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [**not** ] My change requires changes to the documentation.
- [**not** ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1274"></a>

### CODE REVIEW

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by nathgoutam93 — APPROVED** [review]

File: public/data/alokVerma749.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/alokVerma749",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/alok_std",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/alokVerma749.json:18]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

File: public/data/alokVerma749.json (hunk 1 above)
Comment (alokVerma749) [reply public/data/alokVerma749.json]: ok its done

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**